medusa/databases/main_db.py

```diff
@@ -346,26 +346,28 @@ def inc_major_version(self):
         minor_version = 0
         self.connection.action('UPDATE db_version SET db_version = ?, db_minor_version = ?;',
                                [major_version, minor_version])
+        log.info(u'[MAIN-DB] Updated major version to: {}.{}', *self.connection.version)
+
         return self.connection.version
 
     def inc_minor_version(self):
         major_version, minor_version = self.connection.version
         minor_version += 1
         self.connection.action('UPDATE db_version SET db_version = ?, db_minor_version = ?;',
                                [major_version, minor_version])
+        log.info(u'[MAIN-DB] Updated minor version to: {}.{}', *self.connection.version)
+
         return self.connection.version
 
     def execute(self):
-        utils.backup_database(self.connection.path, self.checkDBVersion())
+        utils.backup_database(self.connection.path, self.checkMajorDBVersion())
```

Comment:
I don't these (the backup calls) should be changed.
Don't you need the tuple so that the database backup contains the minor version too?
![image](https://user-images.githubusercontent.com/10238474/47656929-c9aefb00-db98-11e8-9fb6-2385aa78b962.png)

Reply:
I think that's okay, because these backups are going to be of very old versions (< 42) and won't have a minor version anyway. The new versions are using `self.connection.version` which contains the major and minor version.